### ISSUE TITLE
Homogenize icon sizes

### ISSUE THREAD

**Issue by pgilfernandez** [olive-editor/olive#341]
Hi,

The icon set has been designed as 64x64px (or proportional sizes as SVG allow for rendering at your will) so it would be nice if all the GUI has this in mind, just to have a solid and homogenized UI.

I found some cases where icons are bigger:
<img width="466" alt="effects" src="https://user-images.githubusercontent.com/5942369/51462723-a17f0480-1d61-11e9-8e3f-c64949933451.png">
Or smaller:
<img width="723" alt="graph" src="https://user-images.githubusercontent.com/5942369/51462732-a6dc4f00-1d61-11e9-8677-ea0c9bad6310.png">
In this last case, I would say that not only these icons should be regular size but they may be placed close to the "keyframes navigator" buttons. You will save a line and they will fit closer to its related buttons.

Cheers

**Comment by itsmattkc** [comment]
Those are certainly some large icons. This seems to be a Mac-exclusive issue, seemingly stemming from Qt reporting incorrect DPI scale values to Olive. In 963e46af99dc58c065e41faddda20c5b75d4699e, I've made all icon sizes relative to their defaults, hopefully this solves this issue.

Moving the value sliders to the top is probably a good idea, I gave them their own line preparing for an effect row with several fields, but in practice it's probably unlikely there will be more than 3 in any given row.
